CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG for ASAB WebUI Components
 
+## 25.2.6
+
+- Update `Credentials` it can be used as a wrapper (#14)
```

Comment:
@aringocode Try to make the changelog messages clear and understandable: `Refactor the Credentials component so that it can be externally wrapped in a custom component, allowing its inner rendering to be customized` is a bit better

Reply:
Please also update the description of this MR (remove `credentialsToString` from title, add proper description)